python/lsst/ts/imsim/imsimCmpt.py

```diff
@@ -564,10 +564,11 @@ def _mapOpdToZk(self, rotOpdInDeg, numOpd):
 
             # Rotate OPD if needed
             if rotOpdInDeg != 0:
-                opdRot = opd
+                opdRot = opd.copy()
+                minValue = np.nanmin(np.abs(opdRot))
```

Comment:
Could you just add a comment here explaining the need for the minValue parameter?

Reply:
Sorry I missed this! Just added the comment